Jenkinsfile

```diff
@@ -465,46 +479,69 @@ try {
     parallel 'Python2: CPU': {
       node('mxnetlinux-cpu') {
         ws('workspace/ut-python2-cpu') {
-          init_git()
-          unpack_lib('cpu')
-          python2_ut('ubuntu_cpu')
+          try {
+            init_git()
+            unpack_lib('cpu')
+            python2_ut('ubuntu_cpu')
+          } finally {
+            collect_test_results_unix('nosetests1.xml', 'nosetests_python2_cpu1.xml')
+            collect_test_results_unix('nosetests2.xml', 'nosetests_python2_cpu2.xml')
+            collect_test_results_unix('nosetests3.xml', 'nosetests_python2_cpu3.xml')
+          }
         }
       }
     },
     'Python3: CPU': {
       node('mxnetlinux-cpu') {
         ws('workspace/ut-python3-cpu') {
-          init_git()
-          unpack_lib('cpu')
-          python3_ut('ubuntu_cpu')
+          try {
+            init_git()
+            unpack_lib('cpu')
+            python3_ut('ubuntu_cpu')
+          } finally {
+            collect_test_results_unix('nosetests1.xml', 'nosetests_python3_cpu1.xml')
```

Comment:
are these xml files accessible from web UI? how are they going to be presented?

Reply:
They will be available in the web UI like here: http://jenkins.mxnet-ci.amazon-ml.com/job/incubator-mxnet/view/change-requests/job/PR-11199/8/